build.sbt

```diff
@@ -43,6 +43,8 @@ val beamVersion = "2.11.0"
 val enumeratumVersion = "1.5.13"
 val logbackVersion = "1.2.3"
 val scioVersion = "0.7.4"
+val tototoshiVersion = "1.3.5"
```

Comment:
Would rename this to `scalaCsvVersion`, so it describes the library instead of the author.